### PR TITLE
mp3val: modernize + test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/mp3val.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  url "https://downloads.sourceforge.net/mp3val/mp3val-0.1.8-src.tar.gz"
+  sha1 "19f7506d387f72def2861ec271c5cb4135bd8f5e"
 
   def install
     system "gnumake -f Makefile.gcc"
     bin.install "mp3val.exe" => "mp3val"
   end
+
+  test do
+    `#{bin}/mp3val -f #{test_fixtures("test.mp3")}` =~ /Done!$/
```
Comment (jacknagel) [inline Library/Formula/mp3val.rb]: I think this should be something along the lines of 

```rb
assert_match /Done!$/, shell_output("#{bin}/mp3val -f #{test_fixtures("test.mp3")}")
```

Moving forward, explicit assertions are the way to go.

Comment (bfontaine) [inline Library/Formula/mp3val.rb]: Thanks @jacknagel, that’s fixed.

File: Library/Formula/mp3val.rb [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 class Mp3val < Formula
-  homepage 'http://mp3val.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/mp3val/mp3val-0.1.8-src.tar.gz'
-  sha1 '19f7506d387f72def2861ec271c5cb4135bd8f5e'
+  homepage "http://mp3val.sourceforge.net/"
+  url "https://downloads.sourceforge.net/mp3val/mp3val-0.1.8-src.tar.gz"
+  sha1 "19f7506d387f72def2861ec271c5cb4135bd8f5e"
 
   def install
     system "gnumake -f Makefile.gcc"
```
Comment (mikemcquaid) [inline Library/Formula/mp3val.rb]: Can you separate the arguments here? Thanks!

Comment (bfontaine) [inline Library/Formula/mp3val.rb]: @mikemcquaid Thanks, fixed. `brew audit --strict` missed that one, see #35459.

**Pull request closed without merging**